clusterloader2/pkg/measurement/common/slos/network_programming.go

```diff
@@ -47,6 +47,14 @@ func init() {
 
 type netProgGatherer struct{}
 
+func (n *netProgGatherer) IsEnabled(config *measurement.MeasurementConfig) bool {
+	if config.CloudProvider == "kubemark" {
+		klog.Warningf("%s is disabled in Kubemark", n)
```

Comment:
I would remove this warning - logging twice that something is disabled doen't make sense to me.